server/controllers/finance/conventions.js

```diff
@@ -0,0 +1,196 @@
+/**
+* The /cash/conventions HTTP API endpoint
+*
+* @module finance/conventions
+*
+* @desc This module is responsible for handling all crud operations relatives
+* to convention payment and define all convention api functions
+*
+* @required q
+* @required lib/db
+* @required lib/guid
+* @required lib/util
+* @required ledger
+* @required journal/primarycash
+*
+* @todo implements functions for the GET method
+* @todo implements functions for the PUT method
+*/
+
+'use strict';
+
+var q                  = require('q'),
+    db                 = require('../../lib/db'),
+    uuid               = require('../../lib/guid'),
+    util               = require('../../lib/util'),
+    ledger             = require('./ledger'),
+    journalConvention  = require('./journal/primarycash').convention;
+
+var getDebitorReceipt  = ledger.debitor_group;
```

Comment:
We've unfortunately used the name `Debitor` in many places.  It should really be spelled `Debtor`.